vizro-core/tests/integration/test_examples.py

```diff
@@ -48,6 +48,7 @@ def dashboard(request, monkeypatch):
 @pytest.mark.filterwarnings("ignore:HTTPResponse.getheader()")
 # Ignore as it doesn't affect the test run
 @pytest.mark.filterwarnings("ignore::pytest.PytestUnhandledThreadExceptionWarning")
+@pytest.mark.filterwarnings("ignore::pytest.PytestUnraisableExceptionWarning")
```

Comment:
What kind of `PytestUnraisableExceptionWarning` errors we got?

Reply:
Like this
https://github.com/mckinsey/vizro/actions/runs/11703449240/job/32594160883

Reply:
Do you think we can safely ignore it, it seems very unrelated to actual dashboard workings

Reply:
seems because the svg file sometimes not properly closed?
```
ResourceWarning: unclosed file <_io.BufferedReader name='/home/runner/work/vizro/vizro/vizro-core/examples/scratch_dev/assets/logo_light.svg'>
```

Reply:
It warns us that because of unclosed file we can have some memory leak, etc.
To me it looks safe as it sometimes fail on this error and doesn't affect the tests itself.

Reply:
Something like:
```
ignore:unclosed file:ResourceWarning
```